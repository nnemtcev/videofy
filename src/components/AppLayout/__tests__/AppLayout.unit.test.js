import { shallow } from "enzyme";
import React from "react";
import { AppLayout } from "../AppLayout";

describe("AppLayout", () => {
  test("AppLayout renders with no children", () => {
    const wrapper = shallow(<AppLayout />);
    expect(wrapper).toMatchSnapshot();
  });

  test("AppLayout renders with one child", () => {
    const wrapper = shallow(
      <AppLayout>
        <div>Child 1</div>
      </AppLayout>
    );
    expect(wrapper).toMatchSnapshot();
  });

  test("AppLayout renders with multiple children", () => {
    const wrapper = shallow(
      <AppLayout>
        <div>Child 1</div>
        <div>
          <span>Child 2</span>
        </div>
      </AppLayout>
    );
    expect(wrapper).toMatchSnapshot();
  });
});
