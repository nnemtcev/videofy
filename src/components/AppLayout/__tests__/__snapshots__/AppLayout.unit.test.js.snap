// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppLayout AppLayout renders with multiple children 1`] = `
<div
  className="app-layout"
>
  <HeaderNav />
  <div>
    Child 1
  </div>
  <div>
    <span>
      Child 2
    </span>
  </div>
</div>
`;

exports[`AppLayout AppLayout renders with no children 1`] = `
<div
  className="app-layout"
>
  <HeaderNav />
</div>
`;

exports[`AppLayout AppLayout renders with one child 1`] = `
<div
  className="app-layout"
>
  <HeaderNav />
  <div>
    Child 1
  </div>
</div>
`;
